lib/utils/assertThrowsAsync.js

```diff
@@ -4,8 +4,9 @@ const assert = require('assert');
  * Helper for testing async functions. Utilizes original node assert.throws.
  * @param {Function} fn async function
  * @param {RegExp|Function|Object} regExp error
+ * @param {string} error msg
```

Comment:
@AntonSBilan I think the correct jsdoc would look like `* @param {string} msg message for the assertion`